content/hardware/required/thrusters/index.md

```diff
@@ -0,0 +1,42 @@
++++
+title = "Thrusters"
+description = "Used to maneuver a vehicle in marine environments. The number and orientation of thrusters on a vehicle determines the degrees of freedom (DoF) it may maneuver in."
+date = 2022-10-11T17:33:19+11:00
+template = "docs/page.html"
+sort_by = "weight"
+weight = 30
+draft = false
+[extra]
+lead = ""
+toc = true
+top = false
++++
+
+Thrusters are necessary to maneuver an underwater vehicle around. The number and orientation of thrusters on a vehicle determines the number of degrees of freedom (DoF) it may maneuver in.
+
+The maximum current draw of the power supply on the vehicle is an important consideration when choosing what type and how many thrusters to mount on a vehicle. The maximum current draw at the intended voltage should be totaled up for all the thrusters. If this exceeds the current rating of the power supply, either lower the supply voltage, [limit the maximum thrust](https://discuss.bluerobotics.com/t/raspberry-and-pixhawk-shutdown-problem/10814/2), or remove thrusters.
+
+Although brushless or brushed thrusters may be used with ArduSub, they may not be mixed, and the thruster type needs to be paired to an appropriate [Electronic Speed Controller (ESC)](../esc/).
```

Comment:
> they may not be mixed 

why?

Reply:
Looked into this and it seems like there's a [`MOT_PWM_TYPE`](https://www.ardusub.com/developers/full-parameter-list.html#motpwmtype-output-pwm-type) parameter that presumably affects all the "MOTOR" designated outputs. It would be more correct to specify that all motors for vehicle control must support the same type of signal. I'll fix it :-)

Reply:
well, those wont work well with sub, as you cant control direction without an h-bridge...